tests/run

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -e
+
+DOCKER_COMPOSE_CMD="docker-compose -f ./tests/docker-compose.yml"
+MAX_DOCKER_WAIT_SECONDS=60
+
+function waitForPiholeToBeReady () {
+    attempt=${1:-1} # set to 1 if not already set
+    if [ $attempt -gt $MAX_DOCKER_WAIT_SECONDS ]; then
+        echo "Failed waiting on pihole to start...."
+        stopPihole
+        exit 1
+    fi
+    image_status=$(docker inspect -f "{{.State.Health.Status}}" tests-dev-pihole-1 || echo 'missing')
+    if [ "${image_status}" != "healthy" ]; then
+        sleep 1;
+        echo -n "."
+        waitForPiholeToBeReady $((attempt+1))
+    fi
+}
+
+function startPihole() {
+    stopPihole # ensure not lingering images
+    ${DOCKER_COMPOSE_CMD} pull
+    ${DOCKER_COMPOSE_CMD} up -d dev-pihole
+}
+
+function stopPihole() {
+    ${DOCKER_COMPOSE_CMD} down --remove-orphans
+}
+
+function prepForTests() {
+    ./tests/npm install &
+    waitForPiholeToBeReady
+    wait # wait on npm install to finish
+}
+
+startPihole
+prepForTests
+./tests/npm run test:integration
```

Comment:
run the actual tests after the Pihole docker image is healthy and npm install is complete